api/src/index.node.ts

```diff
@@ -67,26 +81,13 @@ const watchDir = process.env.FPX_WATCH_DIR ?? process.cwd();
 startRouteProbeWatcher(watchDir);
 
 // Set up websocket server
-const wss = new WebSocketServer({ server, path: "/ws" });
-wss.on("connection", (ws) => {
-  logger.debug("WebSocket connection established", ws.OPEN);
-  wsConnections.add(ws);
+setupRealtimeService({ server, path: "/ws", wsConnections });
 
-  ws.on("ping", () => {
-    logger.debug("ping");
-    ws.send("pong");
-  });
-  ws.on("error", (err) => {
-    if ("code" in err && err.code === "EADDRINUSE") {
-      logger.error(
-        "WebSocket error: Address in use. Please choose a different port.",
-      );
-    } else {
-      logger.error("WebSocket error:", err);
-    }
-  });
-  ws.on("close", (code) => {
-    wsConnections.delete(ws);
-    logger.debug("WebSocket connection closed", code);
-  });
-});
+// check settings if proxy requests is enabled
+const proxyRequestsEnabled = await getSetting(db, "proxyRequestsEnabled");
+
+if (proxyRequestsEnabled) {
+  logger.info("Proxy requests feature enabled, connecting to webhonc...");
```

Comment:
```suggestion
  logger.debug("Proxy requests feature enabled, connecting to webhonc...");
```

`info` will log to the user's terminal fwiw, probably want debug here?

Reply:
to some extent we'd want to inform the user that `fpx` is making an outside connection but this is kind of evident in the UI so changing it to `debug`